include/flexi_cfg/config/trace-internal.h

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) 2024 Jo Voordeckers
+// Copyright (c) 2020-2022 Dr. Colin Hirsch and Daniel Frey
```

Comment:
This is 95% the `trace.hpp` from pegtl, just changed a few colors and added printouts of the input and the  function to do the nested parse